mbs_results/validation_checks.py

```diff
@@ -53,6 +59,40 @@ def colnames_clash(
     )
 
 
+def period_and_reference_not_given(
+    reference, period, responses_keep_cols, contributors_keep_cols, **config
+):
+    """
+    _summary_
+
+    reference: Str
+      the name of the reference column
+    period: Str
+      the name of the period column
+    response_keep_cols: Str
+      the names of the columns to keep from the responses data
+    contributors_keep_cols: Str
+        the names of the columns to keep from the contributors data
+    **config: Dict
+      main pipeline configuration. Can be used to input the entire config dictionary
+
+    Returns
+    -------
+    bool
+        Returns True if response and period are not in both responses_keep_cols
+        and contributors_keep_cols
+        False otherwise
+    """
+
+    return any(
+        [
```

Comment:
i would rather specifying what `i` is, i understand in programming i refers to the loop element, but i think would help if instead of `i` using something like `col` or `columns` ( it looks look like it refers to columns sorry i am mistaken, but i am happy keeping it as it is

Reply:
Updated loop elements to have better names